test/distributed/test_store.py

```diff
@@ -110,6 +110,18 @@ def _test_compare_set(self, store):
     def test_compare_set(self):
         self._test_compare_set(self._create_store())
 
+    def assertRaisesRegex(self, i, j):
```

Comment:
i think there should be an python unittest function, so you dont need to define it yourself

Reply:
Done, thanks